### DISCUSSION TITLE
How can I delete my library?

### DISCUSSION THREAD

**Discussion by beedell-roke** [Kurozora/kurozora-web#391] [Q&A] [answered]
I imported my library to test its ability to accurately convert the Kitsu export into Kurozora entries, but it's outdated now (due to https://github.com/Kurozora/kurozora-web/discussions/390#discussioncomment-8983483) so I'd like to delete it so that I can properly reimport it.

**Comment by kiritokatklian** [comment, marked as answer]
@RokeJulianLockhart you can select the override option when importing which will delete your library then import the selected file.

There's currently no other option to delete everything in your library with the press of a single button, but it sounds like a handy option to have in such cases. I'll try to add it by tomorrow.

**Reply by beedell-roke** [reply]
Thanks.

**Reply by kiritokatklian** [reply]
[Deleting library settings](https://github.com/Kurozora/kurozora-web/commit/d4194f64287757ebd1736098811634b5c751a61d) has been added. I'll update the website tomorrow, so it shows up for everyone.
